libuavcan/include/libuavcan/libuavcan.hpp

```diff
@@ -8,6 +8,27 @@
  */
 #include "libuavcan/build_config.hpp"
 
+#define __STDC_FORMAT_MACROS
```

Comment:
This is a weird disagreement between ISO C and C++. It's not supposed to be required by C++ but since it is by C there are (reportedly) some C++ compilers out there that require it.

Reply:
okay. removed.